recipe/conda_build_config.yaml

```
@@ -22,31 +22,38 @@ channel_targets:
 # match vcver / vsyear / vsver / runtime_version / cl_version from
 # https://github.com/conda-forge/vc-feedstock/blob/main/recipe/conda_build_config.yaml
 VCVER:
+  - 14.3
   - 14.3
   - 14.2
 VSYEAR:
+  - 2022
   - 2022
   - 2019
 VSVER:
+  - 17
   - 17
   - 16
 RUNTIME_VERSION:
+  - 14.41.34120
   - 14.40.33810
   - 14.29.30139
 CL_VERSION:
+  - 19.41.34120
   - 19.40.33808
   - 19.29.30139
 
 # toolchain name in Microsoft.VisualStudio.Component.VC.<TOOLCHAIN_COMBINED>.x86.x64, see
 # https://learn.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-build-tools
 TOOLCHAIN_COMBINED:
+  - 14.41.17.11
   - 14.40.17.10
   - 14.29.16.11
 
 # header version corresponding to minor version (i.e. x in VS 15.x), see last digits of _MSC_FULL_VER in
 # https://en.wikipedia.org/wiki/Microsoft_Visual_C%2B%2B#Internal_version_numbering;
 # alternatively, download & inspect the *.vsix binaries (for details see install-pkg.sh)
 MSVC_HEADERS_VERSION:
+  - 14.40.33807
```

See https://github.com/conda-forge/clang-win-activation-feedstock/pull/44/commits/79097fad61dea6e8bce865b6b2f7869c8171762b
```suggestion
  - 14.41.34120
```